SECURITY.md

```diff
@@ -0,0 +1,14 @@
+# Security Policy
+
+## Supported Versions
+
+Our smart contracts are under active development. As such, we only support the most current release.
```

Comment:
Should we also mention that we support code that is currently deployed? Because it's often different to main branch

Reply:
I clarified the wording. We'll probably want to update this once @barrutko publishes his BBBP spec.